examples/stable/mysql/kanister/mysql-blueprint.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: cr.kanister.io/v1alpha1
```

Comment:
Let's remove `kanister` from the path.

Reply:
@tdmanv I didn't get you. If we remove `kanister` from apiVersion, it will not work.